src/app/caesar-portal/common/models/location.ts

```diff
@@ -1,4 +1,4 @@
 export class Location {
```

Comment:
btw if you don't have any logic in the class you can define the interface and use it as a type where you need it the rest will be handled by ts